README.md

```diff
@@ -265,8 +264,8 @@ import "github.com/gregjones/httpcache"
     ).WithAuthToken(os.Getenv("GITHUB_TOKEN"))
 ```
 
-Learn more about GitHub conditional requests at
-https://docs.github.com/en/rest/using-the-rest-api/best-practices-for-using-the-rest-api?apiVersion=2022-11-28#use-conditional-requests-if-appropriate
+Learn more about GitHub conditional requests in the
+["Use conditional requests if appropriate"](https://docs.github.com/en/rest/using-the-rest-api/best-practices-for-using-the-rest-api?apiVersion=2022-11-28#use-conditional-requests-if-appropriate).
```

Comment:
Same here... the phrase `in the "Use conditional requests if appropriate"` sounds odd to me.